[2024-04-17T18:21:50.174+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: files_ext.create_directory manual__2024-04-17T18:21:45.868719+00:00 [queued]>
[2024-04-17T18:21:50.184+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: files_ext.create_directory manual__2024-04-17T18:21:45.868719+00:00 [queued]>
[2024-04-17T18:21:50.184+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-17T18:21:50.196+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): create_directory> on 2024-04-17 18:21:45.868719+00:00
[2024-04-17T18:21:50.202+0000] {standard_task_runner.py:60} INFO - Started process 2708 to run task
[2024-04-17T18:21:50.204+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'files_ext', 'create_directory', 'manual__2024-04-17T18:21:45.868719+00:00', '--job-id', '2224', '--raw', '--subdir', 'DAGS_FOLDER/files_extraction.py', '--cfg-path', '/tmp/tmp9btbnhrs']
[2024-04-17T18:21:50.206+0000] {standard_task_runner.py:88} INFO - Job 2224: Subtask create_directory
[2024-04-17T18:21:50.239+0000] {task_command.py:423} INFO - Running <TaskInstance: files_ext.create_directory manual__2024-04-17T18:21:45.868719+00:00 [running]> on host fdd3cfdf3f53
[2024-04-17T18:21:50.301+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='files_ext' AIRFLOW_CTX_TASK_ID='create_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-04-17T18:21:45.868719+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-17T18:21:45.868719+00:00'
[2024-04-17T18:21:50.303+0000] {logging_mixin.py:188} INFO - Directory already exists.
[2024-04-17T18:21:50.303+0000] {python.py:202} INFO - Done. Returned value was: ['dags', 'logs', 'data', 'destination', 'plugins', 'config', 'webserver_config.py', 'extraction_done', '***.cfg']
[2024-04-17T18:21:50.321+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=files_ext, task_id=create_directory, execution_date=20240417T182145, start_date=20240417T182150, end_date=20240417T182150
[2024-04-17T18:21:50.336+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-17T18:21:50.362+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
